docs/Using-Fleet/FAQ.md

```diff
@@ -394,8 +396,16 @@ When updating agent options, you may see an error similar to this:
 If you’re not using the latest osquery, use the fleetctl apply --force command to override validation.
 ```
 
-This error indicates that you're providing a config option that isn't valid in the current version of osquery, typically because you're setting a command line flag through the configuration key. This has always been unsupported through the config plugin, but osquery has recently become more opinionated and Fleet now validates the configuration to make sure there aren't errors in the osquery agent. 
+This error indicates that you're providing a config option that isn't valid in the current version of osquery, typically because you're setting a command line flag through the configuration key. This has always been unsupported through the config plugin, but osquery has recently become more opinionated and Fleet now validates the configuration to make sure there aren't errors in the osquery agent.
 
 If you are not using the latest version of osquery, you can create a config YAML file and apply it with `fleetctl` using the `--force` flag to override the validation:
 
 ```fleetctl apply --force -f config.yaml```
+
+## How can I renew my Apple Business Manager server token?
```

Comment:
@noahtalerman  I took a shot at documenting the Apple BM token renewal here in the FAQ (task 4 of https://github.com/fleetdm/fleet/issues/8725). Let me know if that works for you.

Reply:
@mna looks good to me! 

This made me realize that we're not linking to this FAQ entry from the output of the `get mdm-apple-bm` command (like we do for `get apple-mdm`).

I think it makes sense to update the warning and error messages to the following. What do you think?

```
WARNING: Your Apple Business Manager (ABM) server token is less than 30 days from expiration. If it expires, laptops newly purchased via ABM will not automatically enroll in Fleet. To renew your ABM server token, follow these instructions: https://fleetdm.com/docs/using-fleet/faq#how-can-i-renew-my-apple-business-manager-server-token
```

```
ERROR: Your Apple Business Manager (ABM) server token is less than 30 days from expiration. If it expires, laptops newly purchased via ABM will not automatically enroll in Fleet. To renew your ABM server token, follow these instructions: https://fleetdm.com/docs/using-fleet/faq#how-can-i-renew-my-apple-business-manager-server-token
```

Reply:
@noahtalerman yes, sounds good to me, I'm just starting work on `get mdm-apple-bm` so I'll make sure to use that message format (I'll update them in the ticket).

Reply:
@noahtalerman one thing, the error message reads:

```
ERROR: Your Apple Business Manager (ABM) server token is less than 30 days from expiration. If it expires, ...
```

but in this case, it is already expired, it is not less than 30 days from expiration. I propose this:

```
ERROR: Your Apple Business Manager (ABM) server token is expired. Laptops newly purchased via ABM will not automatically enroll in Fleet. To renew your ABM server token, follow these instructions: https://fleetdm.com/docs/using-fleet/faq#how-can-i-renew-my-apple-business-manager-server-token
```

Reply:
Great catch. Your proposal makes sense to me 👍